thorlcr/activities/funnel/thfunnelslave.cpp

```diff
@@ -293,7 +297,9 @@ class CParallelFunnel : implements IRowStream, public CSimpleInterface
     {
         if (exception)
             throw exception.getClear();
+        BlockedActivityTimer timer(activity.getActivityTimerAccumulator(), activity.queryTimeActivities());
         OwnedConstThorRow row = rows.dequeue();
+        timer.leave();
```

Comment:
this blocked time will be the time blocked because the input-handlers haven't started or kept up and pushed rows in time. 
I think this is correct, with lookahead time tracking the time it spent for input, that time (that will be added to totaltime) 
 - blocked time is any time it spent dawdling waiting for the input handlers to catch up and push more.

I don't think the leave() mechanism is useful here though (and would be better to be removed until/if it really is).
It would better to scope the BlockedActivityTimer as normal.